pom.xml

```diff
@@ -0,0 +1,621 @@
+
+  <developers>
+    <developer>
+      <name></name>
+      <email></email>
+      <organization></organization>
+      <organizationUrl></organizationUrl>
+    </developer>
+  </developers>
+
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <issueManagement>
+    <system>GitHub</system>
+    <url>https://github.com/google/closure-templates/issues</url>
+  </issueManagement>
+
+  <mailingLists>
+    <mailingList>
+      <name>closure-templates-discuss</name>
+      <subscribe>closure-templates-discuss+subscribe@googlegroups.com</subscribe>
+      <unsubscribe>closure-templates-discuss+unsubscribe@googlegroups.com</unsubscribe>
+      <post>closure-templates-discuss@googlegroups.com</post>
+      <archive>https://groups.google.com/forum/#!forum/closure-templates-discuss</archive>
+    </mailingList>
+  </mailingLists>
+
+  <scm>
+    <connection>scm:git:https://github.com/google/closure-templates.git</connection>
+    <developerConnection>scm:git:git@github.com:google/closure-templates.git</developerConnection>
+    <url>https://github.com/google/closure-templates.git</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <java.version>1.7</java.version>
+    <guice.version>3.0</guice.version>
+    <soy.examples>${project.basedir}/examples</soy.examples>
+    <soy.examples.out>target/examples</soy.examples.out>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
+      <version>1</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.inject</groupId>
+      <artifactId>guice</artifactId>
+      <version>${guice.version}</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.inject.extensions</groupId>
+      <artifactId>guice-assistedinject</artifactId>
+      <version>${guice.version}</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.inject.extensions</groupId>
+      <artifactId>guice-multibindings</artifactId>
+      <version>${guice.version}</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>14.0</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.ibm.icu</groupId>
+      <artifactId>icu4j</artifactId>
+      <version>51.1</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>aopalliance</groupId>
+      <artifactId>aopalliance</artifactId>
```

Comment:
This is a transitive dependency of Guice, it's not used directly by the code so better leave it out.